src/core/parse.js

```diff
@@ -43,7 +43,8 @@ export let parse = (obj, selector) => {
                       })
                     : key
             );
-        } else if (val != undefined) {
+        } else if (val + 1) {
```

Comment:
`undefined + 1 = NaN` but `null + 1 = 1`. So you can use this trick to handle null values.

Reply:
Nice!